client/src/components/Menu.tsx

```diff
@@ -57,13 +74,27 @@ export const Menu = () => {
                             gap: '16px',
                         }}
                     >
-                        {links.map(link => (
+                        {links.filter(link => isLinkAnchor(link.to)).map(link => (
                             <li key={link.title}>
-                                <Link to={link.to} style={activeLink === link.to ? activeLinkStyle : linkStyle}>
+                                <Link to={link.to} style={activeLink === link.to ? activeLinkStyle : linkStyle } onClick={clearDropdown}>
                                     {link.title}
                                 </Link>
                             </li>
                         ))}
+                        <FormControl sx={{minWidth: 150, maxHeight: 0, marginTop: -1}} size="small">
+                            <InputLabel id="dropdown-link-select- label" style={{color: 'black'}}>My Data</InputLabel>
+                            <Select 
+                                labelId="dropdown-link-select-label"
+                                id="dropdown-link-select"
+                                value={currentPage}
+                                label="MyData"
+                                onChange={handleChange}
+                            >
+                                {links.filter(link => !isLinkAnchor(link.to)).map(link => (
```

Comment:
We probably want to change the styling for the drop down
Having an outline is a bit odd e.g.
you can remove that by use 
 sx={{ boxShadow: 'none', '.MuiOutlinedInput-notchedOutline': { border: 0 } }}